src/utils/frame.ts

```diff
@@ -0,0 +1,128 @@
+import calculateCrc16 from './calculateCrc16.js';
+import getBytesFromHex from './getBytesFromHex.js';
+import getBytesFromBase64 from './getBytesFromBase64.js';
+import BinaryBuffer from './BinaryBuffer.js';
+
+
+const STUFFING_BYTE = 0x7d;
+
+const byteStuff = ( byte: number ): number => {
+    const byteStuffMap: Record<number, number> = {0x13: 0x33, 0x11: 0x31, 0x7d: 0x5d, 0x7e: 0x5e};
```

Comment:
it's better to declare it outside this function as now this is recreating on each byteStuff call

Reply:
fixed